### ISSUE TITLE
Rake task to implement Creator ordering on legacy items

### ISSUE THREAD

**Issue by vantuyls** [osulp/Scholars-Archive#1584]
### Descriptive summary

Need a rake task to convert legacy creators into nested creators.

- [x] Create lookup table from DSpace with Handle, Creators, and timestamp or other indicator of creator order
- [x] Rake task should look at DSpace table and compare against whats in Hyrax now

- [x] Flag items for which there are discrepancies (see #1583) and create a table of these that we can review

**Comment by zhang4952** [comment]
Here is the summary of my meeting with Greg:
- all DSpace items (handles) will have `dc.creator`, and milti-creators will be separated by '||' like `Forest Products Laboratory (U.S.)||Peck, Edward C. (Edward Coit), 1894-||Selbo, M. L. (Magnus Leonard), 1901-`; https://dspace-ir.library.oregonstate.edu/xmlui/handle/1957/61933?show=full
- an extra field of `dc.identifier.citation` could be available for journal articles like https://dspace-ir.library.oregonstate.edu/xmlui/handle/1957/61933?show=full
- I will provide a spreadsheet with three columns for all handles in DSpace: `handle`, `dc.creator`, `dc.identifier.citation`

**Comment by revgum** [comment]
Here is an example record and the expected format of the CSV from DSpace. @zhang4952 @straleyb 

```
handle_id,metdata_value_id,place,text_value
http://hdl.handle.net/1957/42668,123456,1,"Smit, Ellen"
http://hdl.handle.net/1957/42668,123457,2,"Winters-Stone, Kerrie M."
http://hdl.handle.net/1957/42668,123458,3,"Loprinzi, Paul D."
http://hdl.handle.net/1957/42668,123459,4,"Tang, Alice M."
http://hdl.handle.net/1957/42668,123460,5,"Crespo, Carlos J."
```

**Comment by luisgreg99** [comment]
rake task implemented on https://github.com/osulp/Scholars-Archive/pull/1589

**Comment by zhang4952** [comment]
I attach the link to the creator spreadsheet created for this rake: https://docs.google.com/spreadsheets/d/1FFNx7m9nODR5cmv_JZnbMA6DjlOjHMCHTnT9y1dXCaI/edit?usp=sharing

**Comment by revgum** [comment]
This is being covered in #1640

**Comment by revgum** [comment]
fixed in #1640
